Library/Formula/tmate.rb

```diff
@@ -0,0 +1,37 @@
+class Tmate < Formula
+  desc "Instant terminal sharing"
+  homepage "http://tmate.io"
+  url "https://github.com/tmate-io/tmate/archive/2.2.0.tar.gz"
+  sha256 "932148b24d9c67e524ce744c9480b5603c9f976afb6ad7ca6d63246cd9e7fd3a"
+
+  head "https://github.com/tmate-io/tmate.git"
+
+  bottle do
+    cellar :any
+    root_url "https://github.com/tmate-io/tmate.io/raw/gh-pages/cellar"
+    sha256 "15cd75a8cab65bd5ede8f329d94b1a561c2f44bb73fbcc25e7dd7474ad42fa90" => :el_capitan
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "autoconf"   => :build
```

Comment:
Cut the equal spacing here. It looks a bit neater but a lot of editors flag it and it's not worth that hassle.